### PR TITLE
Hbase new version 2.4.17

### PR DESCRIPTION
**Pull request by maltesander**
# Description

- Added version 2.4.17
- Update to jmx-exporter 0.20.0
- Add nexus upload script

@lfrancke will do more refactoring here. No review required yet!

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
- [ ] Changes are OpenShift compatible
- [ ] All added packages (via microdnf or otherwise) have a comment on why they are added
- [ ] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [ ] All packages should have (if available) signatures/hashes verified
- [ ] Does your change affect an SBOM? Make sure to update all SBOMs
```

### CODE REVIEW

**Comment by lfrancke** [comment]
Is this missing the changes to the Dockerfile?

**Comment by maltesander** [comment]
> Is this missing the changes to the Dockerfile?

Yeah did not check that in somehow

File: hbase/Dockerfile [hunk 1]
```diff
@@ -34,9 +34,10 @@ WORKDIR /stackable
 
 RUN curl -L https://repo.stackable.tech/repository/packages/hbase/hbase-${PRODUCT}-bin.tar.gz | tar -xzC . && \
     ln -s /stackable/hbase-${PRODUCT} /stackable/hbase && \
-    curl https://repo.stackable.tech/repository/packages/jmx-exporter/jmx_prometheus_javaagent-0.16.1.jar \
-    -o /stackable/jmx/jmx_prometheus_javaagent-0.16.1.jar && \
-    chmod -x /stackable/jmx/jmx_prometheus_javaagent-0.16.1.jar && \
+    curl https://repo.stackable.tech/repository/packages/jmx-exporter/jmx_prometheus_javaagent-${JMX_EXPORTER}.jar \
```
[the remark this answers is not in the document]
Comment (maltesander) [reply hbase/Dockerfile:37]: Will do, should i keep the legacy jmx exporter soft link as well (0.16.1)?

[the remark this answers is not in the document]
Comment (lfrancke) [reply hbase/Dockerfile:37]: Ah! Yeah...might make sense, then we can update all operators together after 23.11

**Comment by maltesander** [comment]
Closing in favor of https://github.com/stackabletech/docker-images/pull/494

**Pull request closed without merging**